pylinalg/vector.py

```diff
@@ -105,17 +105,28 @@ def vec_transform(
         transformed vectors
     """
 
-    matrix = np.asarray(matrix)
+    matrix = np.asarray(matrix, dtype="f8")
```

Comment:
I'm curious: is there a reason for using `"f8"` instead of `float`?

Reply:
Yes! They have the same effect, but `"f8"` is a local string literal and `float` needs to be looked up from globals. It improves performance (significantly).

Reply:
wow